src/ai/backend/manager/api/scaling_group.py

```diff
@@ -93,13 +94,29 @@ async def get_wsproxy_version(request: web.Request, params: Any) -> web.Response
                 if not wsproxy_addr:
                     wsproxy_version = "v1"
                 else:
-                    wsproxy_status = await query_wsproxy_status(wsproxy_addr)
-                    wsproxy_version = wsproxy_status["api_version"]
-                return web.json_response(
-                    {
-                        "wsproxy_version": wsproxy_version,
-                    }
-                )
+                    try:
+                        wsproxy_status = await query_wsproxy_status(wsproxy_addr)
+                        wsproxy_version = wsproxy_status["api_version"]
+                        return web.json_response(
+                            {
+                                "wsproxy_version": wsproxy_version,
+                            }
+                        )
+                    except aiohttp.ClientConnectorError:
+                        log.error(
+                            "scaling group({}) uses INVALID WSProxy address: ({})",
```

Comment:
It's an alraedy error message. There is no need to capitalize the description.
Also, let's distinguish the validation of wsproxy addr formats and actual availability of the wsproxy addr.

```suggestion
                            "Failed to query the wsproxy {1} configured for sg:{1}",
```

Reply:
Thanks for reviewing!
I committed.